src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -399,6 +404,61 @@ public IActionResult Shutdown([FromBody] bool corsProtection = true)
             return this.Ok();
         }
 
+        /// <summary>
+        /// Changes the log levels for the specified loggers.
+        /// </summary>
+        /// <param name="request">The request containing the loggers to modify.</param>
+        /// <returns><see cref="OkResult"/></returns>
+        [HttpPut]
+        [Route("loglevels")]
+        public IActionResult UpdateLogLevel([FromBody] LogRulesRequest request)
+        {
+            Guard.NotNull(request, nameof(request));
+
+            // Checks the request is valid.
+            if (!this.ModelState.IsValid)
+            {
+                return ModelStateErrors.BuildErrorResponse(this.ModelState);
+            }
+
+            try
+            {
+                foreach (LogRuleRequest logRuleRequest in request.LogRules)
+                {
+                    LogLevel nLogLevel = logRuleRequest.LogLevel.ToNLogLevel();
+                    LoggingRule rule = LogManager.Configuration.LoggingRules.SingleOrDefault(r => r.LoggerNamePattern == logRuleRequest.RuleName);
+
+                    if (rule == null)
+                    {
+                        throw new Exception($"Logger name `{logRuleRequest.RuleName}` doesn't exist.");
+                    }
+
+                    // Log level ordinals go from 1 to 6 (trace to fatal).
+                    // When we set a log level, we enable every log level above and disable all the ones below.
+                    foreach (LogLevel level in LogLevel.AllLoggingLevels)
+                    {
+                        if (level.Ordinal >= nLogLevel.Ordinal)
+                        {
+                            rule.EnableLoggingForLevel(level);
+                        }
+                        else
+                        {
+                            rule.DisableLoggingForLevel(level);
+                        }
+                    }
+                }
+
+                // Only update the loggers if the setting was succesfull.
```

Comment:
`successful`

Reply:
eh? I changed it yesterday
![image](https://user-images.githubusercontent.com/1867877/52623379-14345900-2ea4-11e9-881c-3d7d7f844f2b.png)

where is it?

Reply:
ah look it's up there ⏫ 
I forgot to push it, but when I did just now, it appeared before your comment. 
I guess the sorting is done by time.

Reply:
I guess github is being weird (or I am)